web/styles/app_components.css

```diff
@@ -970,7 +970,10 @@ div.overlay {
         }
 
         .overlay_message_controls {
-            display: inline-block;
+            display: inline-flex;
```

Comment:
You are right this appears to work when we set `display: flex`. I think since the whole structure is maintained using a grid it kind of makes  `flex` and `inline-flex` behave in a similar fashion. 

I will change it to `display: flex` just to keep it simple.